src/run/docker-compose.ts

```diff
@@ -44,7 +44,7 @@ export class DockerCompose {
         const date = new Date(Date.now());
 
         this.logFile = join(logsDir,
-            `compose.out.${date.getFullYear()}-${date.getMonth()}-${date.getDay()}.txt`);
+            `compose.out.${date.getFullYear()}-${date.getMonth()}-${date.getDate()}.txt`);
```

Comment:
Would it be preferable to have a consistent length of 2 digits even for Months & Days which are of 1 only ?
Having a look, [toLocaleDateString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) should give the result:
```
`compose.out.${date.toLocaleDateString('en-CA', {
  year: 'numeric',
  month: '2-digit',
  day: '2-digit'
})}.txt`);
```
The locale ['en-CA'](https://en.wikipedia.org/wiki/Date_and_time_notation_in_Canada) is one which uses the ISO format YYYY-MM-DD

Reply:
Good idea 👍🏼 I have adopted this into the code now.